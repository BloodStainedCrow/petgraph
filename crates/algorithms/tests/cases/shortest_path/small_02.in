version https://git-lfs.github.com/spec/v1
oid sha256:27c35f6fcc19c459e0efd8c3b22b69e598bba4ea14eac9e72682b06a1d2c6e8a
size 855
