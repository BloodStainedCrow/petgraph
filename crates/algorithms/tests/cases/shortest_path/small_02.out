version https://git-lfs.github.com/spec/v1
oid sha256:ad694a5746d6f0073c119b361ec87b4a790ac8fa48f37404b4ad81f71ee6f609
size 18
